### PR TITLE
[tools/onert_train] Run nnfw_train

### PR DESCRIPTION
**Pull request by jyoungyun**
This commit trains a model by the number of epochs. One epoch performs repeated operations as much as the total data divided by batch_size, and each for loop trains the model using the nnfw_train function.

ONE-DCO-1.0-Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

Draft PR: #10900

### CODE REVIEW

**Comment by jyoungyun** [comment]
@nnfw-bot test onert-x64-release

File: tests/tools/onert_train/src/onert_train.cc [hunk 1]
```diff
@@ -180,38 +180,44 @@ int main(const int argc, char **argv)
...
-        [&](const benchmark::Phase &phase, uint32_t nth) {
-          std::cout << "... "
-                    << "run " << nth + 1 << " takes " << phase.time[nth] / 1e3 << " ms"
-                    << std::endl;
-        },
-        1, true);
+      for (uint32_t n = 0; n < num_sample; ++n)
+      {
+        // get batchsize data
+        if (!generator(n, input_data, expected_data))
```
Comment (glistening) [inline tests/tools/onert_train/src/onert_train.cc:190]: @jyoungyun I have no idea what `generator` generate. I found

https://github.com/Samsung/ONE/blob/52430200a2aa3b79b8ed53e2eac70f003ccabaeb/tests/tools/onert_train/src/rawdataloader.h#L31-L33

But it provides no information. Could you please explain me what `generator` generate?

**Review by glistening — APPROVED** [review]
LGTM

**Review by ragmani — APPROVED** [review]
LGTM

**Pull request merged**